seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1816,12 +1816,14 @@
         "  }",
         "  ",
         "  .red-box {",
-        "    background-color: red;",
+        "    background-color: #be0041;",
```

Comment:
These copy changes all look great. I'm just wondering why you changed the color from "red" to "#be0041". Red is obvious to any human reading it, where as "#be0041" is unknown and may distract or intimidate learners.

Reply:
I see that you did this for color contrast rules. Could you find another named color that has sufficient contrast and change it for that throughout the lessons? I'd rather work with named CSS colors here, since it's not the section that talks about hex colors.

Reply:
Ah yes, that makes sense. Although, the class name does indicate that the colour is red. But I agree, less is more here.